app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
```

Comment:
- [ ] [OPTIONAL]
Most buttons within the app appear to be just plain text and do not have any UX effect on them. Let's endeavor to improve upon this aspect by ensuring that all buttons have some features of normal button, such as border, background color or mouse hover effects so as to improve the user experience.

![image](https://github.com/lRebornsl/recipe-app/assets/12745474/5203162e-0550-4f3d-ade0-004c70e21546)

Reply:
@rloterh Thanks for the feedback! 👍🏼 

Changes applied 😄